### PR TITLE
Collocation endpoints

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**
Adds collocation results and summary endpoints.

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

**Comment by Codebmk** [comment]
The status field has true as the output yet we are expecting either passed, scheduled, failed or running
image

**Comment by NoahNsimbe** [comment]
> All is working fine save device collocation results Screenshot from 2023-03-06 16-09-32

Done @Codebmk

File: src/device-monitoring/helpers/utils.py [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+
+    try:
+        token = auth_value.split(" ")[1]
+        decoded = jwt.decode(token, verify=False)
+        user_details["first_name"] = decoded.get("firstName", "")
+        user_details["last_name"] = decoded.get("lastName", "")
+        user_details["email"] = decoded.get("email", "")
+        user_details["id"] = decoded.get("_id", "")
+    except:
+        pass
```
Comment (NoahNsimbe) [inline src/device-monitoring/helpers/utils.py:29]: @Baalmart using JWT token to get details of the user who made the request.

**Review by Codebmk — APPROVED** [review]
LGTM thanks

**Pull request merged**